client/src/components/header/InvitationsButton.vue

```diff
@@ -83,27 +84,48 @@ async function updateInvitations(): Promise<void> {
   }
 }
 
-async function openInvitationsPopover(event: Event): Promise<void> {
+async function openInvitationsMenu(event: Event): Promise<void> {
   event.stopPropagation();
-  const popover = await popoverController.create({
-    component: InvitationsListPopover,
-    alignment: 'center',
-    event: event,
-    cssClass: 'invitations-list-popover',
-    showBackdrop: false,
-    componentProps: {
-      informationManager: informationManager,
-    },
-  });
-  await popover.present();
-  const { role, data } = await popover.onDidDismiss();
-  await popover.dismiss();
-  if (role === MsModalResult.Confirm && data.action) {
-    if (data.action === InvitationAction.Greet) {
-      await greetUser(data.invitation);
-    } else if (data.action === InvitationAction.Cancel) {
-      await cancelUserInvitation(data.invitation);
-    } else if (data.action === InvitationAction.Invite) {
+  let result: { data?: { action: InvitationAction; invitation?: UserInvitation }; role?: string };
+
+  if (isLargeDisplay.value) {
+    const popover = await popoverController.create({
+      component: InvitationsListPopover,
+      alignment: 'center',
+      event: event,
+      cssClass: 'invitations-list-popover',
+      showBackdrop: false,
+      componentProps: {
+        informationManager: informationManager,
+      },
+    });
+    await popover.present();
+    result = await popover.onDidDismiss();
+    await popover.dismiss();
+  } else {
+    const modal = await modalController.create({
+      component: InvitationsListModal,
+      cssClass: 'invitations-list-modal',
+      showBackdrop: true,
+      handle: false,
+      breakpoints: isLargeDisplay.value ? undefined : [1],
```

Comment:
I guess a good rule of thumb for sheet-modals would be to add a 0 breakpoint if we don't set backdropDismiss to false. So the user can dismiss the modal by clicking elsewhere or swiping down.